CRM/Contact/BAO/Relationship.php

```diff
@@ -295,33 +295,38 @@ public static function add($params, $ids = [], $contactId = NULL) {
     if ($params['id']) {
       $hook = 'edit';
     }
-    //@todo hook are called from create and add - remove one
```

Comment:
This comment is still true if coming from legacyCreateMultiple which is still used in some places. I'm not suggesting to deal with that now, just that there is still a todo.

Reply:
ok - I've moved the comment to where it belongs in `legacyCreateMultiple()`